contrib/README.md

```
@@ -38,3 +38,9 @@ Utilities to generate test vectors for the data-driven Dash tests.
 
 ### [Verify-Binaries](/contrib/verify-binaries) ###
 This script attempts to download and verify the signature file SHA256SUMS.asc from bitcoin.org.
+
+Command Line Tools
+---------------------
+
+### [Completions](/contrib/completions) ###
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: setext; Actual: atx_closed

(MD003, heading-style)